modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-test/bnd.bnd

```diff
@@ -1,4 +1,8 @@
 Bundle-Name: ${artifactId}-test
 Bundle-SymbolicName: ${package}.test
 Bundle-Version: ${version}
--includeresource: ${package}.client.jar=${artifactId}-client-*.jar;lib:=true
\ No newline at end of file
+#if (${buildType} != "gradle")
+-includeresource: ${package}.client.jar=../${artifactId}-client/target/${artifactId}-client-${version}.jar;lib:=true
```

Comment:
I know we're adding the support for maven, but does the *-test.jar work in maven?

Reply:
If it doesn't work even if deployed, maybe we should remove it using the postGenerate task.

Reply:
The truth is that we don't know if it is possible to run this kind of tests with Maven. The only reason to keep this folder in Maven is to ease a change in the build mechanism to gradle, but, as this seems uncommon, the best option may be to don't generate this module with Maven

Reply:
I was thinking about adding a flag to control if the user want to generate client and test modules or not. I'll try to add it and consider it in the generation tests